test/functional/test_framework/blocktools.py

```diff
@@ -52,6 +52,9 @@
 # Genesis block time (regtest)
 TIME_GENESIS_BLOCK = 1296688602
 
+# Coinbase transaction outputs can only be spent after this number of new blocks (network rule)
```

Comment:
There is a definition you can also borrow elements from in test/functional/interface_bitcoin_cli.py:
```text
# The block reward of coinbaseoutput.nValue (50) BTC/block matures after
# COINBASE_MATURITY (100) blocks. Therefore, after mining 101 blocks we expect
# node 0 to have a balance of (BLOCKS - COINBASE_MATURITY) * 50 BTC/block.
```

Reply:
Thank you for bringing my attention to this file - I like the comment. Anyway, I think the current comment is reasonably good. If you think it can/should be improved, please suggest a better comment and I'll (most likely) accept it :)